feat!: update to Capacitor 7

gabekole opened the pull request:
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [/ ] The changes have been tested successfully.

gabekole committed:
Update to capacitor 7

tolutronics commented:
Can someone approve this please

bjornmassoels commented:
+1

Zwarmex commented:
+1 too

valentinbld45 commented:
Don't want to spam, but @robingenz / @jcesarmobile is there's anything we can do to help?

robingenz commented:
Please fix the build and format your code so that we can merge this.

Craic-T commented:
+1

thammerl committed:
Use Java 21 in CI build

thammerl committed:
Fix code formatting

thammerl commented:
@gabekole I created [pull request for your fork](https://github.com/gabekole/speech-recognition/pull/1) to fix the issues with the build and the code formatting.

thammerl committed:
Use arm64 runner in GitHub workflow

thammerl committed:
Try to fix CI build

 - `package-lock.json` is missing @rollup/rollup-darwin-arm64.
 - See https://github.com/npm/cli/issues/4828.

gabekole committed:
Merge pull request #1 from thammerl/master

Upgrade CI build to Java 21 and fix code formatting

gabekole commented:
Working on my repo now.

robingenz commented on package.json:
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@capacitor-community/speech-recognition",
-  "version": "6.0.1",
+  "version": "7.0.0",
```
Please revert this. The version is bumped on release.

robingenz commented:
@gabekole Have you tested your changes?

gabekole committed:
Revert version  number change

robingenz reviewed (commented):
Thank you!

tolutronics commented:
Thanks Guys!

bjornmassoels commented:
yaaaaay! thank you
